modules/javascript-udfs/pages/using-n1ql-inside-javascript-udfs.adoc

```diff
@@ -0,0 +1,145 @@
+= Using N1QL inside JavaScript UDFs
```

Comment:
I'd be tempted to change this to "embedding" &mdash; I wasn't sure what the difference was between this page and "Calling N1QL from JavaScript", but the word "embedding" in the abstract made it clear

```suggestion
= Embedding N1QL in JavaScript UDFs
```